content/_index.md

```diff
@@ -17,6 +17,24 @@ kube-vip provides Kubernetes clusters with a virtual IP and load balancer for bo
 
 [![Build and publish main image regularly](https://github.com/kube-vip/kube-vip/actions/workflows/main.yaml/badge.svg)](https://github.com/kube-vip/kube-vip/actions/workflows/main.yaml)
 
+## Modes 
+
+### ARP
+
+**ARP** is a layer 2 protocol that is used to inform the network of the location of a new address. When a new IP address is configured to a device, there needs to be a mechanism to inform the network of which piece of hardware is hosting this new address. ARP is the technology that is used to ensure that a network understands the link between the hardware address (MAC) and the logical address (IP).
```

Comment:
Should it mention Gratuitous Arp? This definition comes off a little misleading.